### ISSUE TITLE
ECWeather Type Error

### ISSUE THREAD

**Issue by JwKoroluk** [michaeldavie/env_canada#83]
Hi, small issue. Looks like the data source for wind speed isn't just integers but includes strings like 'Calm'. 

```
File /local_disk0/.ephemeral_nfs/cluster_libraries/python/lib/python3.9/site-packages/env_canada/ec_weather.py:485, in ECWeather.update(self)
    476 # Update hourly forecasts
    477 for f in weather_tree.findall("./hourlyForecastGroup/hourlyForecast"):
    478     self.hourly_forecasts.append(
    479         {
    480             "period": parse_timestamp(f.attrib.get("dateTimeUTC")),
    481             "condition": f.findtext("./condition"),
    482             "temperature": int(f.findtext("./temperature") or 0),
    483             "icon_code": f.findtext("./iconCode"),
    484             "precip_probability": int(f.findtext("./lop") or "0"),
--> 485             "wind_speed": int(f.findtext("./wind/speed") or 0),
    486             "wind_direction": f.findtext("./wind/direction"),
    487         }
    488     )

ValueError: invalid literal for int() with base 10: 'Calm'
```

**Comment by sam-rostum** [comment]
I can second this bug report and also confirm that the error is very repeatable.

I have included logs from the Environment Canada integration in Home Assistant (which imports env_canada ec_weather.py) as snipped below:

>> File "/usr/local/lib/python3.11/site-packages/env_canada/ec_weather.py", line 485, in update
>>    "wind_speed": int(f.findtext("./wind/speed") or 0),
>>    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>> ValueError: invalid literal for int() with base 10: 'Calm'

Although I would certainly agree that mixing actual speeds and string descriptors as EC appears to be doing is a terrible idea, nevertheless the library shouldn't lose its marbles when "Calm" gets returned as a wind speed.

**Comment by michaeldavie** [comment]
Fixed by #84
